CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [1.13.3]
```

Comment:
Is it `1.12.3`?

Reply:
thanks Victor, I also noticed it just after I'd submitted :D